audit_engine/core/engine.py

```
@@ -0,0 +1,442 @@
+            
+            for adapter in self.static_adapters:
+                for contract_path in contract_paths:
+                    future = executor.submit(self._run_static_adapter, adapter, contract_path)
+                    futures.append((adapter, future))
+            
+            # Collect results with timeout
+            for adapter, future in futures:
+                try:
+                    adapter_findings = future.result(timeout=request.max_analysis_time)
+                    findings.extend(adapter_findings)
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)